_validate/addonVersion_schema.json

```
@@ -357,13 +368,15 @@
 				{
 					"language": "de",
 					"displayName": "Mein Addon",
-					"description": "erleichtert das Durchführen von xyz"
+					"description": "erleichtert das Durchführen von xyz",
+					"changelog": "Neue Funktionen"
 				}
 			],
 			"required": [
 				"language",
 				"displayName",
-				"description"
+				"description",
+				"changelog"
```

As suggested in https://github.com/nvaccess/addon-datastore-validation/pull/48/files#r2377541352

```suggestion
				"description"
```